degica/komoju-woocommerce
# Optionally generate gateways for each individual payment type

This PR adds an interface for dynamically creating WC payment gateways for each KOMOJU payment type.

https://user-images.githubusercontent.com/2793160/169770488-21e9fa39-bf09-46dc-ac0c-2f4eface57b6.mp4

## How to test

See [docs/dev_setup.md](https://github.com/degica/komoju-woocommerce/blob/master/docs/dev_setup.md) for setup instructions.

1. Go to woocommerce settings
2. Click the KOMOJU tab
3. Make sure you have a valid secret key
4. Check some payment method boxes and save
5. Go to the "Payments" tab
6. You should see gateways for the payment methods you checked. Enable them.
7. Go to your storefront and purchase an item
8. You should be able to pay with one of the single-method gateways
